unbxdAutosuggest.js

```diff
@@ -637,21 +642,21 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, params) {
 			if (this.activeRow >= 0 && this.activeRow < lis.size()) {
 				this.$input.val($(lis[this.activeRow]).data('value'));
 				if (this.options.filtered && this.activeColumn === 0) {
-					var dataValue = $(lis[this.activeRow]).attr('data-value') ? $(lis[this.activeRow]).attr('data-value') : '';
+					var dataValue = $(lis[this.activeRow]).attr('data-value') ? $(lis[this.activeRow]).attr('data-value').replace('amp;', '') : '';
 					var dataFiltername = $(lis[this.activeRow]).attr('data-filtername') ? $(lis[this.activeRow]).attr('data-filtername') : '';
 					var dataFiltervalue = $(lis[this.activeRow]).attr('data-filtervalue') ? $(lis[this.activeRow]).attr('data-filtervalue') : '';
 					var query = dataValue + (dataFiltername != '' ? ':' + dataFiltername + ':' + dataFiltervalue : '')
 					// updating product header while hovering on suggestions
-					if (this.options.filtered && this.options.popularProducts.header) {
+					if (this.options && this.options.popularProducts.header) {
```

Comment:
why we removed `filtered`

Reply:
parent level `If` condition is checking for `this.options.filtered` so here If we make sure options & popularProducts pass the condition that will be enough